frontend/src/App.vue

```diff
@@ -382,6 +382,9 @@ export default {
     async displayGeneratorsPrompt() {
       const uiOptions = await this.rpc.invoke("getState");
       this.messages = uiOptions.messages;
+      if (this.isInVsCode()) {
+        window.vscode.setState(uiOptions);
+      }
       await this.rpc.invoke("receiveIsWebviewReady", []);
```

Comment:
1. rename func setMessages() to setState()
    async setState() {
      const uiOptions = await this.rpc.invoke("getState");
      this.messages = uiOptions.messages;
      if (this.isInVsCode()) {
        window.vscode.setState(uiOptions);
      }
    }
2. 
    async displayGeneratorsPrompt() {
      await this.setState();
      await this.rpc.invoke("receiveIsWebviewReady", []);
    },

Reply:
Done